lms/migrations/versions/9f1e88367596_email_unsubscribe.py

```diff
@@ -0,0 +1,35 @@
+"""
+Add the email_unsubscribe table.
+
+Revision ID: 9f1e88367596
+Revises: f3d631c110bf
+Create Date: 2023-04-06 14:44:40.077112
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "9f1e88367596"
+down_revision = "f3d631c110bf"
+
+
+def upgrade():
+    op.create_table(
+        "email_unsubscribe",
+        sa.Column(
+            "created", sa.DateTime(), server_default=sa.text("now()"), nullable=False
+        ),
+        sa.Column(
+            "updated", sa.DateTime(), server_default=sa.text("now()"), nullable=False
+        ),
+        sa.Column("id", sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column("tag", sa.UnicodeText(), nullable=False),
```

Comment:
Not adding any DB level constrain for this for now. They are a bit of a pain to maintain and we can rely on the code test for the right values here.

Reply:
Yeah it's fine :man_shrugging: I don't think it's super important to avoid invalid values slipping into this table anyway